docs/index.md

```diff
@@ -142,6 +148,32 @@ The URL at `<<iss value from Signed JWT>>` SHALL NOT include a trailing `/`. For
 }
 ```
 
+### Certificates
+
+X.509 certificates can be used by issuers to indicate the issuer's participation in a PKI-based trust framework.
+
+If the Verifier supports PKI-based trust frameworks and the Health Card issuer includes the `"x5c"` parameter in matching JWK entries from the `.keys[]` array,
+the Verifier establishes that the issuer is trusted as follows:
+1. Verifier validates the leaf certificate's binding to the Health Card issuer by:
```

Comment:
Perhaps this a product of my not being as versed in the field, but I find the "leaf" language a bit confusion. Can we reference an RFC if it's a well-known term? If not, can we tweak the language? Are we just referring to the encoded certificate that the `x5c` value?

Reply:
The underlying structure is: a verifier trusts some "trust framework root CA", in the form of a certificate containing the root CA's; this root CA is used to sign some "chain" of intermediate certificates which ends in a cert containing the issuer key, etc:

    Trusted Root Cert --> ["Intermediate" Certs]{0, *} --> ["Leaf" Cert w/ key from  $issuer/.well-known/jwks.json]

So the JWK in the jwks.json file points to a *chain* of certificates from the Leaf up to the Root. The verifier needs to walk up this chain and ensure that signatures are valid... all the way to the top, and also ensure that the root represents a trusted entity, and that the leaf represents the issuer of the SMART Health Card being validated.

There's some background in https://tools.ietf.org/html/rfc5280#section-4.2.1.7 which could be a good reference. Perhaps there are better references?

Reply:
> There's some background in https://tools.ietf.org/html/rfc5280#section-4.2.1.7 which could be a good reference. Perhaps there are better references?

Good reference; [section 3.2](https://tools.ietf.org/html/rfc5280#section-3.2) might be a better starting point.

Reply:
This is really helpful - thanks for clarifying!